components/ppp_link/ppp_link.c

```diff
@@ -134,8 +137,6 @@ esp_err_t ppp_link_init(const ppp_link_config_t *_config) {
     ESP_ERROR_CHECK(uart_set_pin(config.uart, config.io.tx, config.io.rx,
                            config.io.rts, config.io.cts));
 
-    ESP_ERROR_CHECK(uart_set_hw_flow_ctrl(config.uart, UART_HW_FLOWCTRL_CTS_RTS, UART_FIFO_LEN - 8));
```

Comment:
if (config.flow_control == UART_HW_FLOWCTRL_CTS_RTS)
    ESP_ERROR_CHECK(uart_set_hw_flow_ctrl(config.uart, UART_HW_FLOWCTRL_CTS_RTS, UART_FIFO_LEN - 8));

Reply:
uart_param_config() does that for you. That was why I moved the hardcoded flow control settings to main instead.